javascript/javascript3/week3/RewriteTime.js

```diff
@@ -0,0 +1,25 @@
+//1.
+
+function promiseToTimeOut(milisecs) {

```

Comment:
Not sure if there is a reason to mix arrow functions with usual ones (I would use only arrow ones 🙂)